### PR TITLE
Agile is About Your Turning Radius Blog Post

### PR DESCRIPTION
**Pull request by apmarshall**

### CODE REVIEW

File: _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Agile is About Measuring Your Turning Radius, Not Your 0-60 Start Time"
+short_title: "The Real Meaning of Agile"
```
Comment (BarakStout) [inline _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md:4]: Please remove "Real"

File: _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+But what if all that speed is aimed in the wrong direction?
+
+In the first **Cars** installation, Lightning is challenged by an older car to a race in the desert sand. In a _tortoise and hare like_ fashion, he goes from 0-100+ immediately. Then he hits a curve. Not knowing how to turn on sand, he flies off the track and lands in a cactus patch. His older, slower challenger easily wins the race.
+
+The point of the story isn’t that we should be slow. It’s that you have to know how to turn or your speed will end up wrecking you.
+
+Everyone these days wants to be "agile". Where "agile" is a synonym for "nimble", and being nimble is about how fast you can change directions, not how fast you can move in a straight line. Think about a basketball player who can react quickly to where the defense is setting up and find just the right angle for their pass or shot. That’s what it means to be nimble.
```
Comment (apmarshall) [inline _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md:24]:
```suggestion
Everyone these days wants to be "agile", but "agile" is a synonym for "nimble", and being nimble is about how fast you can change directions, not how fast you can move in a straight line. Think about a basketball player who can react quickly to where the defense is setting up and find just the right angle for their pass or shot. That’s what it means to be nimble.
```

File: _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Agile is About Measuring Your Turning Radius, Not Your 0-60 Start Time"
+short_title: "The Meaning of Agile"
+date: 2022-01-07
```
Comment (BarakStout) [inline _posts/2022-07-02-Agile-Is-About-Measuring-Your-Turning-Radius.md:5]: Needs to update date.

**Review by BarakStout — CHANGES_REQUESTED** [review]

**Comment by apmarshall** [comment]
@BarakStout I believe this is ready for merging. Let me know if there is anything else required.

**Review by BarakStout — APPROVED** [review]
LGTM 👍

**Pull request merged**